apps/internal/base/base.go

```diff
@@ -293,10 +293,6 @@ func (b Client) AcquireTokenSilent(ctx context.Context, silent AcquireTokenSilen
 	authParams.AuthorizationType = silent.AuthorizationType
 	authParams.Claims = silent.Claims
 	authParams.UserAssertion = silent.UserAssertion
-	//if instance discovery is disabled, add authority to known hosts, so that Alias Lookup is not done
-	if authParams.AuthorityInfo.InstanceDiscoveryDisabled {
```

Comment:
What's the benefit of moving this into the storage implementations?

Reply:
I figured it'd be clearer if the alias was added during the flow that required the alias to be in the KnownAuthorityHosts.